### PR TITLE
Host group ID in Action condition => String

### PR DESCRIPTION
**Pull request by kz0817**
[server] Change the type of the Hostgroup ID in Action conditionItem ID to a string.

Some monitoring systems such as OpenStack's ceilometer
use UUIDs to identify some resources. In order to save that
kind of data, Hatohol should also handle them as a string.

### CODE REVIEW

File: server/common/ZabbixAPI.cc [hunk 1]
```diff
@@ -1363,23 +1364,19 @@ void ZabbixAPI::pushTriggersHostid(JSONParser &parser,
 	parser.endObject();
 }
 
+void ZabbixAPI::pushTriggersHostid(JSONParser &parser, ItemGroup *itemGroup)
```
Comment (ashie) [inline server/common/ZabbixAPI.cc]: Using lower case for "id" is mistake?

Comment (kz0817) [inline server/common/ZabbixAPI.cc]: > Using lower case for "id" is mistake?

Yes, It's my mistake. I'll fix it soon.

Comment (kz0817) [inline server/common/ZabbixAPI.cc]: OK. I fixed it. Please continue to review it.

**Comment by ashie** [comment]
It works fine (maybe).

**Pull request merged**